cachito/workers/pkg_managers/general_js.py

```diff
@@ -125,22 +123,21 @@ def download_dependencies(request_id, deps):
                 shutil.move(bundle_dir.npm_deps_dir.joinpath(tarball), dep_dir.joinpath(tarball))
 
 
-def finalize_nexus_for_js_request(request_id):
+def finalize_nexus_for_js_request(repo_name, username):
     """
     Finalize the Nexus configuration so that the request's npm repository is ready for consumption.
 
-    :param int request_id: the ID of the request that Nexus should be configured for
-    :return: the username and password of the Nexus user that has access to the request's npm
-        repository
-    :rtype: (str, str)
+    :param str repo_name: the name of the repository for the request for this package manager
+    :param str username: the username of the user to create for the request for this package manager
```

Comment:
Can `:param str username: the username of the **user to create** for the request for this package manager` be `:param str username: the username of the **user to be created** for the request for this package manager` for better readability?

Reply:
Thanks for the suggestion!